crates/cheatcodes/src/inspector.rs

```diff
@@ -439,8 +440,8 @@ pub struct Cheatcodes {
     /// `char -> (address, pc)`
     pub breakpoints: Breakpoints,
 
-    /// Optional RNG algorithm.
-    rng: Option<StdRng>,
+    /// Optional cheatcodes `TestRunner`.
```

Comment:
Can you document that this is only used for generating random values

Reply:
added in https://github.com/foundry-rs/foundry/pull/8882/commits/0a56dfcf2110f55c08bf5fdc86330970b57c07dc